src/SMAIAXBackend.Application/Services/Implementations/MeasurementListService.cs

```diff
@@ -0,0 +1,29 @@
+using SMAIAXBackend.Application.DTOs;
+using SMAIAXBackend.Application.Exceptions;
+using SMAIAXBackend.Application.Services.Interfaces;
+using SMAIAXBackend.Domain.Model.Entities;
+using SMAIAXBackend.Domain.Model.ValueObjects.Ids;
+using SMAIAXBackend.Domain.Repositories;
+
+namespace SMAIAXBackend.Application.Services.Implementations;
+
+public class MeasurementListService(
+    IMeasurementRepository measurementRepository,
+    ISmartMeterRepository smartMeterRepository) : IMeasurementListService
+{
+    public async Task<List<MeasurementRawDto>> GetFilteredMeasurementsByTenantAndSmartMeterAsync(
```

Comment:
Rename `GetFilteredMeasurementsByTenantAndSmartMeterAsync` to `GetFilteredMeasurementsBySmartMeterAsync`.